### PR TITLE
Duplicate a dashboard and its queries

### LINKED ISSUE #14: Add an example that clones a dashboard and its queries
I already wrote this. Just need to clean it up and migrate it in.

### PR DESCRIPTION
**Pull request by susodapop**
## Type of PR

- [x] Addition

## Description

Adds an example script / entry point that duplicates not only the dashboard but its underlying queries. This PR also applies black formatting to `client.py` for the first time. And adds a new `duplicate_query` method to `client.py` which seems general purpose.

## Related Tickets & Documents

Closes #14

### CODE REVIEW

File: redash_toolbelt/examples/clone_dashboard_and_queries.py
Comment (arikfr) [inline redash_toolbelt/examples/clone_dashboard_and_queries.py:74]: Copy paste...

**Comment by rbval** [comment]
Hi there! Any ETA on when this will be merged? Needed badly.

**Comment by JeffreyYun** [comment]
Also waiting on merge.

Some references to this PR:
https://discuss.redash.io/t/forking-a-dashboard/5376/11
https://discuss.redash.io/t/duplicate-dashboard/1417
Workaround until it's merged:
https://discuss.redash.io/t/forking-a-dashboard/5376/6

**Review by arikfr — APPROVED** [review]

**Pull request merged**
